### PR TITLE
JAMES-1941 Add Content-Length in download response

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

File: server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java [hunk 1]
```diff
@@ -118,6 +118,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             Attachment attachment = attachmentManager.getAttachment(AttachmentId.from(blobId), mailboxSession);
             IOUtils.copy(attachment.getStream(), resp.getOutputStream());
 
+            resp.setContentLength((int) attachment.getSize());
```
Comment (mbaechler) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java]: you can't return more than 2 GiB ?

Comment (aduprat) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java]: No

Comment (mbaechler) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java]: That's a bug !
I can work around it by using :  
`response.setHeader("Content-Length", String.valueOf(attachment.getSize());`

Comment (aduprat) [inline server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java]: Done

**Pull request merged**